packages/flutter_tools/lib/src/project.dart

```diff
@@ -67,6 +67,57 @@ class FlutterProject {
   /// The generated IosModule sub project of this module project.
   IosModuleProject get iosModule => new IosModuleProject(directory.childDirectory('.ios'));
 
+  Future<File> get androidLocalPropertiesFile {
+    return _androidLocalPropertiesFile ??= manifest.then((FlutterManifest manifest) {
+      return directory.childDirectory(manifest.isModule ? '.android' : 'android')
+          .childFile('local.properties');
+    });
+  }
+  Future<File> _androidLocalPropertiesFile;
+
+  Future<File> get generatedXcodePropertiesFile {
+    return _generatedXcodeProperties ??= manifest.then((FlutterManifest manifest) {
+      return directory.childDirectory(manifest.isModule ? '.ios' : 'ios')
+          .childDirectory('Flutter')
+          .childFile('Generated.xcconfig');
+    });
+  }
+  Future<File> _generatedXcodeProperties;
+
+  File get flutterPluginsDotFile {
```

Comment:
This property name seems a bit strange to me, as the `.` is a file name prefix. Maybe remove `Dot`?

Reply:
Done